src/Symfony/Bundle/FrameworkBundle/composer.json

```
@@ -29,6 +29,7 @@
         "symfony/polyfill-mbstring": "~1.0",
         "symfony/filesystem": "^5.4|^6.0",
         "symfony/finder": "^5.4|^6.0",
+        "symfony/secret": "^5.4|^6.0",
```

This should be:
```suggestion
        "symfony/secret": "^6.1",
```
Cause this component will not exists before that release.